gradle-plugins/biz.aQute.bnd.gradle/src/test/groovy/aQute/bnd/gradle/TestBundlePlugin.groovy

```diff
@@ -409,4 +409,28 @@ class TestBundlePlugin extends Specification {
 		result.task(":jar").outcome == UP_TO_DATE
 		result.task(":testOSGi").outcome == UP_TO_DATE
 	}
+
+	def "Bundle Extension added to Jar Task with Local Project Dependency receives jar file (Gradle = #gradleVersion)"() {
+		given:
+		String testProject = "builderplugin5"
+		File testProjectDir = new File(testResources, testProject).canonicalFile
+		assert testProjectDir.isDirectory()
+
+		when:
+		def result = TestHelper.getGradleRunner()
+				.withProjectDir(testProjectDir)
+				.withArguments("--parallel", "--stacktrace", "resolve")
+				.withPluginClasspath()
+				.withDebug(true)
+				.withGradleVersion(gradleVersion)
```

Comment:
I'm not sure if they'll want to run this with multiple versions in an ad-hoc way.  I think this was useful for diagnosing/fixing the original problem in the RC, but I don't think we should include it here.

Reply:
I wanted to provide a test that would demonstrate the change in behavior between Gradle versions if the changes to `BundleTaskExtension` are reverted.  After the fix is accepted and the project updated to build using Gradle 8.2, this could be changed later to just test the latest Gradle version.